plugins/sinks/http/http_test.go

```diff
@@ -154,6 +154,46 @@ func TestSink(t *testing.T) {
 		err = httpSink.Sink(context.TODO(), getExpectedVal(t))
 		assert.NoError(t, err)
 	})
+
+	t.Run("should return no error when using templates", func(t *testing.T) {
```

Comment:
should we consider using no-whitespaces testcases naming?

Reply:
i would keep this to be consistent as existing one, better to have a separate PR for that